src/graphql/data/TopTokens.ts

```diff
@@ -26,24 +34,30 @@ gql`
       address
       symbol
       market(currency: USD) {
+        id
```

Comment:
did you use a lib to add the `id` field or was this done manually?

Reply:
id's are needed for caching/normalization to work, had to do this manually.